third_party/libxml2_config/config.h

```diff
@@ -121,7 +121,9 @@
 #define HAVE_PRINTF 1
 
 /* Define if <pthread.h> is there */
+#if !defined(_WIN32)
```

Comment:
still confused y this has an effect as it shouldn't be included for windows: https://github.com/google/maldoca/blob/master/bazel/libxml.BUILD#L84

Reply:
if u wanna keep it, plz also update: https://github.com/google/maldoca/tree/master/third_party/libxml2_config#readme

Reply:
let's keep it as is for now